x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFrameRestTestCase.java

```diff
@@ -161,7 +161,8 @@ protected void createContinuousPivotReviewsTransform(String transformId, String
 
         String config = "{ \"dest\": {\"index\":\"" + dataFrameIndex + "\"},"
             + " \"source\": {\"index\":\"" + REVIEWS_INDEX_NAME + "\"},"
-            + " \"sync\": {\"time\":{\"field\": \"timestamp\", \"delay\": \"15m\"}},"
+            //Set frequency high for testing
```

Comment:
nit: "Set frequency high" - DYM low? The default is 60s, so 1s is lower?

Reply:
High frequency => low time between repeats

(Similar to high frequency => short wavelength)

But if it's confusing then maybe bypass this by changing the comment to something like `// Set frequency to 1s to make the test run as fast as possible`.

Reply:
This is one of those pedantic things that annoy me; frequency is measured in Hz not seconds, this is an interval. However, most people will read this and understand that frequency represents the period here and is probably the least confusing naming option.

Reply:
A high frequency in Hz means a short distance between cycles. 

A high frequency here means a short time between attempts.